index.html

```diff
@@ -1779,6 +1779,13 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+		<section>
```

Comment:
Not sure we need an entirely new section for this... let's just move the `<p>` into the the description of the member.  Same for the other ones.